Classes/Domain/Entity/Scope.php

```diff
@@ -13,7 +13,7 @@ final class Scope implements ScopeEntityInterface
 {
     use EntityTrait;
 
-    public function jsonSerialize()
+    public function jsonSerialize(): mixed
```

Comment:
`mixed` is not available in PHP 7.4.

I'm also open to drop 7.4-compatibility, since it's EOL anyways.

Reply:
missed that. Remove it, if it's the only php 8 only stuff